java/test/jmri/jmrix/loconet/LnPortControllerTest.java

```diff
@@ -13,6 +15,12 @@
 
     private LocoNetSystemConnectionMemo memo;
 
+    @Test
+    public void testGetAndSetOutputInterval() {
```

Comment:
Shouldn’t this be moved up to the parent test class?  Also, why test this here, since the value is being set on the memo, not the port controller?

Reply:
would have been a remnant from times when AcelaTurnoutManager could not directly access the memo. Fixed since, took out this test here